Update release notes for version 1.5.3 (#1468)
Automattic/studio: RELEASE-NOTES.txt

Unreleased
==========

1.5.3
=====
* Added support to follow symlinks in Previews and Sync #1447, #1448
* Added robust error handling and Sentry tracking for backup extraction #1455
* Fixed Command Prompt button truncation and naming on Windows #1440, #1441
* Fixed default editors when only one is installed #1461
* Improved site export performance #1457
* Removed outdated WP-CLI proof-of-concept code #1426
* Improved "What's New" modal with simplified version check logic #1446
* Improved HTTP request reliability by reducing timeout to 30 seconds #1456
* Improved translation and lint tooling to include `common` folder #1444
* Improved Windows editor path detection by sanitizing regex #1411
* Upgraded Electron Forge and Playground packages #1460

> * Updated Ukrainian and global translations #1462

Reviewer: We constantly update translations for each release, so there is no need to mention that. We can remove Sentry error handling, "What's New" modal version check change, and lint tooling items as well.

Author: Thanks for the feedback. Updated in https://github.com/Automattic/studio/pull/1468/commits/e23faeccd40083a0362258963de2ebd09588d34a.

Unreleased
==========

1.5.3
=====

> * Added support to follow symlinks in Preview Sites and Jetpack Backups 1447, 1448

Reviewer: ```suggestion
* Added support to follow symlinks in Previews and Sync #1447, #1448
```

Author: Thanks @wojtekn for the suggestion. I have updated the notes.

* Added robust error handling and Sentry tracking for backup extraction #1455
* Fixed Command Prompt button truncation and naming on Windows #1440, #1441
* Fixed default editors when only one is installed #1461

> * Fixed backup export performance by removing `getSiteFiles` #1457

Reviewer: ```suggestion
* Improved site export performance #1457
```

> * Fixed outdated WP-CLI proof-of-concept code by removing it #1426

Reviewer: ```suggestion
* Removed outdated WP-CLI proof-of-concept code #1426
```

* Improved "What's New" modal with simplified version check logic #1446
* Improved HTTP request reliability by reducing timeout to 30 seconds #1456
* Improved translation and lint tooling to include `common` folder #1444
* Improved Windows editor path detection by sanitizing regex #1411
* Upgraded Electron Forge to the latest version from the 7.8 branch #1460
* Upgraded Playground packages to 1.0.38 and removed Symlink manager #1425

> * Updated Ukrainian and global translations for version #1462

Reviewer: We could merge dependency updates to one item, and remove some of the items that are invisible to users like improved version check for modal or improving lint tooling.
